docs/DeveloperGuide.md

```diff
@@ -320,6 +320,9 @@ The first 2 operations are exposed in the `Model` interface as `Model#canUndo()`
 
 Given below is an example usage scenario and how the undo mechanism behaves at each step.
 
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:** The `ModelManager**` in the following diagrams is actually stored as `Pair<McGymmy<Pair<Predicate<Food>, MacroList>>`
```

Comment:
Not sure if the `<` works here, @Jh123x can you help me check this

Reply:
I think we need to use `&lt` -> `<` and `&gt` -> `>`

<div markdown="span" class="alert alert-info">
<code>Pair&ltMcGymmy&ltPair&ltPredicate&ltFood&gt, MacroList&gt&gt</code>
</div>

Reply:
Isn't it this part "The `ModelManager**` in the following diagrams"

Reply:
Ohh ok yea